### PR TITLE
Initial Release

### LINKED ISSUE #1: Initial implementation
Initial implementation according to https://github.com/it-at-m/digiwf-ok.ewo-integration

https://address-service-test.muenchen.de/api-docs

Each REST endpoint should be provided as an Kafka listener within the integration service.

### LINKED ISSUE #2: Adress service integration application
To reach the internal LHM address service application within the process via Kafka, an integration artefact has to be implemented.

With the address service you get via GET adresse/search for a given address (here: Marsstr. 22) among other things the following information: 
 "parklizenzgebietId": 35,
 "parklizenzgebietName": "Königsplatz"

For more details see here:

https://address-service-test.muenchen.de/swagger-ui/index.html#/

TODOs:

* Create Repo
* Create maven module for integration logic in repo
* Create starter module using logic module in repo
* Create integration service using starter module to provide it as an image
* Create pipelines to build Repo modules and image

### PR DESCRIPTION
**Pull request by boal**
**Description**
Initial release

**Reference**
Issues #XXX

**Checklist**

- [x] Acceptance criteria are met
- [ ] Pipeline has been run successfully
- [ ] JUnit tests have been run successfully
- [ ] Changelog is adjusted
- [ ] [Documentations](https://git.muenchen.de/groups/digitalisierung/-/wikis/Dokumentationen) are completed
- [ ] Frontend is tested
- [ ] Created sub-branches are deleted
- [ ] Boards are updated ( [Digitalisierung](https://git.muenchen.de/groups/digitalisierung/-/boards) , [Support](https://git.muenchen.de/digitalisierung/digiwf-support) , [Zenhub](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) )
- [ ] feature process is created or extended
- [ ] [git-ops](https://git.muenchen.de/digitalisierung/digiwf-ops) is customized
- [ ] Openshift environments are prepared (Secrets, etc.)

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
```
Comment (lmoesle) [inline .github/workflows/build.yaml]:
````suggestion
      - name: Install Java and Maven
        uses: actions/setup-java@v3.3.0
        with:
          distribution: 'zulu'
          java-version: '11'
````

We should use the same version of the action all workflows

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  release:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Java and Maven
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```
Comment (lmoesle) [inline .github/workflows/release.yaml]:
````suggestion
      - name: Install Java and Maven
        uses: actions/setup-java@v3.3.0
        with:
          distribution: 'zulu'
          java-version: '11'
````

We should use the same version of the action all workflows

File: digiwf-address-service-integration-service/src/main/resources/application-local.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        spring:
+          json:
+            trusted:
+              packages: '*'
+    ssl:
+      key-password: ''
+      key-store-location: ''
+      key-store-password: ''
+      trust-store-location: ''
+      trust-store-password: ''
```
Comment (lmoesle) [inline digiwf-address-service-integration-service/src/main/resources/application-local.yml:55]: You have to activate the `local` profile to enable spring cloud stream. Is that behaviour is intended?

**Review by lmoesle — APPROVED** [review]
👍🏻 well done

File: digiwf-address-service-integration-service/src/main/resources/application-local.yml (hunk 3 above)
Comment (boal) [reply digiwf-address-service-integration-service/src/main/resources/application-local.yml:55]: This are the properties for a local run of the application.

If the application is run an a server or within a container the properties can be set in other ways (e.g. with a openshift secret). So no profile has to be set. Additionally the propeties file is a kind of documentation which properties has to be set to run the application.

**Pull request merged**